File: OnlinePK-Android/biz-live/src/main/java/com/netease/biz_live/yunxin/live/constant/ErrorCode.java
/*
 * Copyright (c) 2021 NetEase, Inc.  All rights reserved.
 * Use of this source code is governed by a MIT license that can be found in the LICENSE file.
 */

package com.netease.biz_live.yunxin.live.constant;

public @interface  ErrorCode {
    /**
     * 断开连接
     */
    int ERROR_CODE_DISCONNECT = 3001;

    /**
     * nertc 被回收
     */
    int ERROR_CODE_ENGINE_NULL = 3002;

    /**
     * 已经接受等待时进入超时
     */
    int ERROR_CODE_TIME_OUT_ACCEPTED = 4001;

    /**
     * 呼叫超时
     */
    int ERROR_CODE_TIME_OUT_CALL_OUT = 4002;

    /**
     * 接收超时
     */
    int ERROR_CODE_TIME_OUT_ACCEPT = 4003;

    /**
     * 状态错误
     */
    int ERROR_CODE_STATE_ERROR = 4004;

    /**
     * 直播频道不存在
     */
    int CREATE_LIVE_NOT_EXIST = 655;
}

File: OnlinePK-Android/lib-network-kt/src/main/java/com/netease/yunxin/lib_network_kt/network/ErrorCode.kt
/*
 *  Copyright (c) 2021 NetEase, Inc.  All rights reserved.
 *  Use of this source code is governed by a MIT license that can be found in the LICENSE file
 */

package com.netease.yunxin.lib_network_kt.network

/**
 * 错误码
 */
object ErrorCode {
    /**
     * 未知错误码
     */
    const val DEFAULT = -1

    /**
     * 返回data为空
     */
    const val ERROR_CODE_EMPTY = 1001
}

object ErrorMsg {
    /**
     * 服务端未知错误
     */
    const val SERVER_UNKNOWN_ERROR = "server unknown error"
}